proto/storage/alert.proto

```diff
@@ -52,6 +53,11 @@ message Alert {
     string namespace_id = 6; // This field has to be duplicated in Alert for scope management and search.
   }
 
+  message Host {
```

Comment:
Ummm, message Host but fields are Node*.  Can we rename to Node?

Reply:
This is another terminology case we need to standardize on. Traditionally, we've used Node everywhere in the code. Node CVEs, Nodes, and hence logical for it to be Node Violations and Node Policies? cc: @mansursyed